### DISCUSSION TITLE
Which 🏀 team has completed two threepeats?

### DISCUSSION THREAD

**Discussion by NomanGhiasy** [ParanoidUser/yolo#1079] [Galaxy Trivia] [answered]
Bonus: Which NBA team has made the most appearances in the finals?

**Comment by peterbaccaro** [comment, marked as answer]
The basketball team that has completed two threepeats is the **Chicago Bulls**. The Chicago Bulls, led by Michael Jordan, achieved a remarkable feat in the 1990s, winning three consecutive NBA championships not once but twice.

The **Los Angeles Lakers** have competed in the NBA Finals the most times, with 32 appearances.

**Reply by NomanGhiasy** [reply]
Correct
